### PR TITLE
bump operator to v1.4.0-beta.1 for release-1.4

### PR DESCRIPTION
**Pull request by WizardXiao**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### What is changed, added, or deleted? (Required)
`find . -type f | xargs sed -i "s/v1.4.0-alpha.1/v1.4.0-beta.1/g"`

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [x] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)
- [ ] v1.1 (TiDB Operator 1.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

File: en/releases/release-1.4.0-alpha.1.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ TiDB Operator version: 1.4.0-alpha.1
 
 ## Rolling Update Changes
 
-- Due to changes in [#4494](https://github.com/pingcap/tidb-operator/pull/4494), if you deploy TiCDC without setting the `log-file` configuration item, upgrading TiDB Operator to v1.4.0-alpha.1 causes TiCDC to rolling upgrade.
+- Due to changes in [#4494](https://github.com/pingcap/tidb-operator/pull/4494), if you deploy TiCDC without setting the `log-file` configuration item, upgrading TiDB Operator to v1.4.0-beta.1 causes TiCDC to rolling upgrade.
```
Comment (ran-huang) [inline en/releases/release-1.4.0-alpha.1.md:17]: Please revert this.

File: zh/releases/release-1.4.0-alpha.1.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ TiDB Operator 版本：1.4.0-alpha.1
 
 ## 滚动升级改动
 
-- 由于 [#4494](https://github.com/pingcap/tidb-operator/pull/4494) 的变更，如果你部署的 TiCDC 没有设置 `log-file` 配置项，那么升级 TiDB Operator 到 v1.4.0-alpha.1 后会导致 TiCDC 滚动重建。
+- 由于 [#4494](https://github.com/pingcap/tidb-operator/pull/4494) 的变更，如果你部署的 TiCDC 没有设置 `log-file` 配置项，那么升级 TiDB Operator 到 v1.4.0-beta.1 后会导致 TiCDC 滚动重建。
```
Comment (ran-huang) [inline zh/releases/release-1.4.0-alpha.1.md:17]: please revert this

File: en/configure-a-tidb-cluster.md [hunk 3]
```diff
@@ -483,7 +483,7 @@ spec:
 > **Note:**
 >
 > - If the TiCDC version is earlier than v6.3.0, TiDB Operator forces an upgrade on TiCDC, which might cause replication latency increase.
-> - The feature is available since TiDB Operator v1.3.8.
+> - The feature is available since TiDB Operator v1.4.0-beta.1.
```
Comment (ran-huang) [inline en/configure-a-tidb-cluster.md:486]: please revert this

File: zh/configure-a-tidb-cluster.md [hunk 4]
```diff
@@ -471,7 +471,7 @@ spec:
 > **注意：**
 >
 > - 如果使用 TiCDC 版本小于 v6.3.0，TiDB Operator 会强制升级 TiCDC，导致同步延时上升。
-> - 该功能自 TiDB Operator v1.3.8 起可用。
+> - 该功能自 TiDB Operator v1.4.0-beta.1 起可用。
```
Comment (ran-huang) [inline zh/configure-a-tidb-cluster.md:474]: revert this

**Review by ran-huang — APPROVED** [review]
Rest LGTM

File: en/releases/release-1.4.0-alpha.1.md (hunk 1 above)
Comment (WizardXiao) [reply en/releases/release-1.4.0-alpha.1.md]: ok

File: zh/releases/release-1.4.0-alpha.1.md (hunk 2 above)
Comment (WizardXiao) [reply zh/releases/release-1.4.0-alpha.1.md]: ok

File: en/configure-a-tidb-cluster.md (hunk 3 above)
Comment (WizardXiao) [reply en/configure-a-tidb-cluster.md]: ok

File: zh/configure-a-tidb-cluster.md (hunk 4 above)
Comment (WizardXiao) [reply zh/configure-a-tidb-cluster.md]: ok

**Review by csuzhangxc — APPROVED** [review]

**Pull request merged**